<?xml version="1.0"?>
<launch>
  <arg name="path_topic" default="/getback_waypoint/getback_waypoint_raw" />
  <arg name="twist_topic" default="/getback_waypoint/cmd_vel" />
  <arg name="map_frame" default="map" />
  <arg name="current_pose_topic" default="/current_pose" />
  <arg name="object_detected_topic" default="/object_detected" />
  <arg name="done_collect_topic" default="/done_collect" />
  <arg name="linear_velocity" default="1.0" />
  <arg name="lookahead_distance" default="0.5" />

  <node name="getback_to_waypoint_node" pkg="getback_to_waypoint" type="getback_to_waypoint_node" output="screen" respawn="true" >
    <param name="path_topic" value="$(arg path_topic)" />
    <param name="twist_topic" value="$(arg twist_topic)" />
    <param name="map_frame" value="$(arg map_frame)" />
    <param name="current_pose_topic" value="$(arg current_pose_topic)" />
    <param name="object_detected_topic" value="$(arg object_detected_topic)" />
    <param name="done_collect_topic" value="$(arg done_collect_topic)" />
    <param name="linear_velocity" value="$(arg linear_velocity)" />
    <param name="lookahead_distance" value="$(arg lookahead_distance)" />
  </node>
</launch>
